src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -124,6 +124,8 @@
     {{ ccjRequestedForClaimantDashboard(claim) }}
   {% elif status === ClaimStatus.REDETERMINATION_BY_JUDGE %}
     {{ redeterminationClaimantDashboard() }}
+  {% elif claim.response.responseMethod == ResponseMethod.OCON_FORM %}
```

Comment:
`ClaimStatus` Is a status that is only used on the frontend (i.e it holds more states that ClaimState on the ClaimStore) and is built in `claim.ts`

We probably should have a new status of `DEFENDANT_OFCOM_RESPONSE` and then follow the pattern of checking this status in this god awful nested if structure. 

This way it would be easier to refactor this if statement to a switch instead which is something that needs to be done in the future.